### PR TITLE
create cate-cli recipe

### PR DESCRIPTION
**Pull request by hans-permana**
Hello! First time submitting a pull request here. I have followed the [step-by-step instructions](https://conda-forge.org/docs/recipe.html#step-by-step-instructions)  as well as the [checklist ](https://conda-forge.org/docs/recipe.html#checklist). 

According to the documentation, upon successful merging, I would be given access to conda-forge repo and the CI builds. Looking forward to it!

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2018, conda-forge
```
Comment (synapticarbors) [inline LICENSE.txt]: Please do not delete this license

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```
Comment (synapticarbors) [inline LICENSE]: This license should go in the recipe directory for `cate-cli`

File: recipes/cate-cli/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,142 @@
+{% set name = "cate-cli" %}
+{% set version = "2.0.0.dev10" %}
+{% set sha256 = "9ad7f8bfbff8ae22788efbc94a66f038f79ad2ee47ba786fdc9e4c9f977f3bc2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CCI-Tools/cate/archive/v{{ version }}.tar.gz
```
Comment (mariusvniekerk) [inline recipes/cate-cli/meta.yaml]: Is it possible to download the other files that are bundled in this recipe as well?  
Conda allows you to specify multiple urls that will get downloaded.

Comment (hans-permana) [reply recipes/cate-cli/meta.yaml]: Hi! Thank you for the review, but I am not clear which other files you referred to? Do you mean the dependencies?

Comment (mariusvniekerk) [reply recipes/cate-cli/meta.yaml]: Things like recipes/cate-cli/cate-cli.app/Contents/Info.plist the icons the pngs etc

**Pull request closed without merging**